### PR TITLE
Add quadlet support for Rootfs and SELinux labels  containers

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
Add Rootfs= support to podman quadlet to allow containers based on rootfs in addition to image.
SELinux label types can not be specified in quadlet, including disabling selinux separation.
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@alexlarsson PTAL

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 1]
```diff
@@ -190,6 +190,14 @@ at `container_gid`.
 
 If `RemapUsers` is enabled and set to `auto`, this specifies the count of the ids to remap
 
+#### `Rootfs=`
+
+The rootfs to use for the container. Rootfs points to a directory on the system that contains the content to be run within the container. This option conflicts with the `Image` option.
+
+The format of the rootfs is the same as when passed to `podman run --rootfs`, so it supports ovelay mounts as well.
```
Comment (LorbusChris) [inline docs/source/markdown/podman-systemd.unit.5.md:197]:
```suggestion
The format of the rootfs is the same as when passed to `podman run --rootfs`, so it supports overlay mounts as well.
```

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 2]
```diff
@@ -190,6 +190,14 @@ at `container_gid`.
 
 If `RemapUsers` is enabled and set to `auto`, this specifies the count of the ids to remap
 
+#### `Rootfs=`
+
+The rootfs to use for the container. Rootfs points to a directory on the system that contains the content to be run within the container. This option conflicts with the `Image` option.
```
Comment (LorbusChris) [inline docs/source/markdown/podman-systemd.unit.5.md:195]:
```suggestion
The root filesystem to use for the container. Rootfs points to a directory on the system that contains the content to be run within the container. This option conflicts with the `Image` option.
```

**Review by LorbusChris — COMMENTED** [review]
one typo and one suggestion

**Comment by rhatdan** [comment]
@ygalblum @vrothberg PTAL

**Comment by alexlarsson** [comment]
lgtm

**Comment by ygalblum** [comment]
I know this OR is already merged, but I did have two nits:
1. Docs: In a different PR, you alphabetically reordered the keys, but here RootFs was added at the wrong place
2. E2E Tests: Should there be a test with both RootFs and Image set?

**Pull request merged**